client/CreatePostForm.tsx

```diff
@@ -2,10 +2,15 @@
 
 import { useRouter } from "next/navigation";
 import { trpc } from "~/client/trpcClient";
+import { rsc } from "~/server-rsc/trpc/client";
```

Comment:
hmmm

```suggestion
import { rsc } from "~/server-rsc/trpc";
```

does this still work?